### PR TITLE
Grid-Overlay module.

### PR DESCRIPTION
**Pull request by Harshithpabbati**
Fixes #[Add issue number here.]

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by Harshithpabbati** [comment]
@jywarren This is a new module which overlays grid over the image.
grid

**Comment by Harshithpabbati** [comment]
@jywarren @publiclab/is-reviewers have a look at this.

**Comment by jywarren** [comment]
This is so nice!!! Is there a parameter we could use to set the grid square
size? Thank you!!!

**Comment by Harshithpabbati** [comment]
Yeah I guess can do it. I will make it and update it.

File: src/modules/GridOverlay/Module.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  
+  var output;
+
+  function draw(input, callback, progressObj) {
+
+    progressObj.stop(true);
+    progressObj.overrideFlag = true;
+
+    var step = this;
+    if (!options.step.inBrowser) { // This module is only for browser
```
Comment (Divy123) [inline src/modules/GridOverlay/Module.js]: Can we use canvas module of node to get this for node as well?

Comment (Harshithpabbati) [reply src/modules/GridOverlay/Module.js]: Sorry I can't get you???

Comment (Divy123) [reply src/modules/GridOverlay/Module.js]: I mean like as of now we are having this module only for browser but we can make it node compatible by using canvas module from npm as well.
Does that make sense?

Comment (Harshithpabbati) [reply src/modules/GridOverlay/Module.js]: Oh fine.

Comment (Harshithpabbati) [reply src/modules/GridOverlay/Module.js]: I will check it out.

Comment (Divy123) [reply src/modules/GridOverlay/Module.js]: Sure please!!

**Comment by Harshithpabbati** [comment]
gridover
@jywarren Done with adding the x and y as inputs.

**Comment by jywarren** [comment]
This is really nice!!!!!! Thanks!

**Comment by Harshithpabbati** [comment]
Do I need to write the doc for the module ?

**Pull request merged**